src/test/parsers/militaryParser.test.js

```diff
@@ -47,3 +48,50 @@ test('set matching text', t => {
   t.equals(results[0].index, 6, 'sets index')
   t.end()
 })
+
+test('military format and date with a year', t => {
+  const results = parser.parse('Aug 11 2017 1100-2030')
+
+  t.equals(results[0].start.get('day'), 11, 'day is the 11th')
+  t.equals(results[0].start.get('month'), 8, 'month is August')
+  t.equals(results[0].start.get('year'), 2017, 'year is 2017')
+
+  t.equals(results[0].start.get('hour'), 11, 'start is 11 am')
+  t.equals(results[0].start.get('minute'), 0, 'start is 11:00 am')
+  t.equals(results[0].end.get('hour'), 20, 'end is 8 pm')
+  t.equals(results[0].end.get('minute'), 30, 'end is 8:30 pm')
+
+  t.end()
+})
+
+test('military format with implied date', t => {
+  const results = parser.parse('Yesterday 1100-2030')
+  const yesterday = moment().subtract(1, 'day')
+
+  t.equals(results[0].start.get('day'), yesterday.date(), 'day is yesterday')
+  t.equals(results[0].start.get('month'), yesterday.month()+1, 'month is current month') // Moment starts the month count at 0 (eg. January = 0)
```

Comment:
Just in case you aren't familiar starting the month at 0 is a JS-wide convention, not just moment. I didn't realize chrono didn't follow this.

Reply:
Ah thanks!